google-cloud-logging/lib/google/cloud/logging/entry/http_request.rb

```diff
@@ -33,7 +33,30 @@ def initialize
           ##
           # The request method. Examples: `"GET"`, `"HEAD"`, `"PUT"`, `"POST"`.
           # (String)
-          attr_accessor :method
+          attr_accessor :request_method
+
+          ##
+          # @overload method()
+          #   Deprecated. Use {#request_method} instead.
+          #
+          #   The request method. Examples: `"GET"`, `"HEAD"`, `"PUT"`,
+          #   `"POST"`. (String)
+          def method *args
+            # Call Object#method when args are present.
+            return super if args.any?
```

Comment:
Should this be `unless args.empty?`?
because `[nil].any? # => false`

Reply:
@dazuma Updated with test coverage.